src/signTransaction.c

```diff
@@ -690,6 +682,7 @@ __noinline_due_to_stack__ void signTx_handleStartDHEncodingAPDU(
                                             SIZEOF(ctx->otherPubkey.W),
                                             ctx->value,
                                             SIZEOF(ctx->value));
+        ASSERT(outlen != 0);
```

Comment:
I generally use `> 0` for uints (e.g. like in bip44.c), it would be best if it was consistent across the code.